gravitee-node-api/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>io.gravitee.node</groupId>
         <artifactId>gravitee-node</artifactId>
-        <version>1.24.3</version>
+        <version>1.25.0-8218-prometheus-labels-SNAPSHOT</version>
```

Comment:
note for later, should we apply the same principle as for apim ? i.e `<version>${revision}${sha1}${changelist}</version>`

Reply:
Don't know what it is about but why not. Maybe something to add to the backlog.